e2e-tests/native/test/Action.ts

```diff
@@ -0,0 +1,74 @@
+type SwipeDirectionType = 'UP' | 'DOWN'
+
+type SwipePoint = { x: number, y: number }
+
+const swipeMove = (startPoint: SwipePoint, endPoint: SwipePoint): Array<{ action: string, options: any }> => {
+    return [
+        {
+            action: 'press',
+            options: startPoint,
+        },
+        {
+            action: 'wait',
+            options: {
+                ms: 200,
+            },
+        },
+        {
+            action: 'moveTo',
+            options: endPoint,
+        },
+        {
+            action: 'release',
+            options: {},
+        }
+    ]
+}
+
+export const swipe = async (direction: SwipeDirectionType, repeat: number = 1, amount: number = 80) => {
+    const {width, height} = await driver.getWindowSize()
+
+    // do a vertical swipe by percentage
+    const upperPercentage = 50 + amount / 2;
+    const lowerPercentage = 50 - amount / 2;
+
+    if (upperPercentage > height || lowerPercentage < 0) {
+        throw new Error('Cannot press beyond the border of the screen.')
+    }
+
+    const anchorPercentage = 50;
+
+    let startPoint: { x: number, y: number };
+    let endPoint: { x: number, y: number };
+
+    switch (direction) {
+        case "UP":
+            startPoint = {
+                x: width * anchorPercentage / 100,
```

Comment:
Couldn't you just do as `anchorPercentage` is always constant, so this would be easier to understand?
```suggestion
                x: width * 0.5,
```

Reply:
Really? If there is just 0.5, i would wonder why it is like this. Think the original solution prevents magic numbers and is more understandable.

Reply:
Could you explain please, I don't get it...

Reply:
Okay, I am stupid. This is just about no magic number and speaking variables, right? And not that the resulting value is not `0.5`...